### PR TITLE
Add documentation page for story points & estimation

### PR DESCRIPTION
**Pull request by rlskoeser**

### CODE REVIEW

File: content/docs/story-points.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+It’s typical to use the Fibonacci sequence for estimates: the bigger something is, the less we know, and it’s not worth quibbling over whether something is an 8 or a 9: it’s just big. An estimate over 5 is usually a sign that the task is too big and should be broken down into smaller parts. 
+
+Story points are used to calculate team **velocity**; the teams' velocity is based on an average of the points for tested and accepted stories over the last three iterations. We use the rolling velocity to determine the appropriate amount of work to take on in the next iteration. Story points should _never_ be used to track individual team member productivity.
+
+{{< figure src="/featured-background.jpg" caption="When we estimate in person, we use planning poker cards. Photo by Nick Budak.">}}
+
+* * * 
+
+For a critical discussion of Agile practices including story poitns and estimation, see [Best Practices?](https://cdh.princeton.edu/updates/2019/07/29/best-practices/) by Rebecca Sutton Koeser. For an alternate estimation scale, see [Estimating Complexity, by Liz Keogh](https://lizkeogh.com/2013/07/21/estimating-complexity/).
```
Comment (kmcelwee) [inline content/docs/story-points.md:15]: `poitns` -> `points`?

File: content/docs/story-points.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Story Points"
+---
+
+Story points are an arbitrary measure of complexity used to calculate and project team velocity. They are not meant to correspond to a measure of time or difficulty. Estimation is typically done as a team, and talking though upcoming features in order to estimate them is a valuable exercise. It gives everyone familiarity with the work that's planned, and gives team members a chance to ask questions and remind each other about the steps involved or potential problems.
+
+It’s typical to use the Fibonacci sequence for estimates: the bigger something is, the less we know, and it’s not worth quibbling over whether something is an 8 or a 9: it’s just big. An estimate over 5 is usually a sign that the task is too big and should be broken down into smaller parts. 
+
+Story points are used to calculate team **velocity**; the teams' velocity is based on an average of the points for tested and accepted stories over the last three iterations. We use the rolling velocity to determine the appropriate amount of work to take on in the next iteration. Story points should _never_ be used to track individual team member productivity.
```
Comment (kmcelwee) [inline content/docs/story-points.md:9]: teams' -> team's ?

And that's great to clarify on using story points to track productivity / performance!

**Comment by rlskoeser** [comment]
@kmcelwee thanks for catching those typos!

File: content/docs/story-points.md (hunk 2 above)
Comment (rlskoeser) [reply content/docs/story-points.md]: > And that's great to clarify on using story points to track productivity / performance!

I think this is one of the ways that the points system sometimes gets misused. That's not what it's intended for, and I thought it worth stating explicitly!

**Comment by rlskoeser** [comment]
I fixed the typos Kevin caught, going to go ahead and merge.

**Pull request merged**